app/controllers/notifications_controller.rb

```diff
@@ -30,14 +30,10 @@ def index
 
       include_reviewables = false
 
-      if SiteSetting.legacy_navigation_menu?
-        notifications = Notification.recent_report(current_user, limit, notification_types)
-      else
-        notifications =
-          Notification.prioritized_list(current_user, count: limit, types: notification_types)
-        # notification_types is blank for the "all notifications" user menu tab
-        include_reviewables = notification_types.blank? && guardian.can_see_review_queue?
-      end
```

Comment:
This condition is `legacy_navigation_menu`, not the notifications menu. So I don't think we want this change?